content/Assets/Styles/settings/_form.scss

```diff
@@ -0,0 +1,26 @@
+/* ==========================================================================
+   #FORM
+   ========================================================================== */
+
+$form-border-size: 2px;
+
+$form-checkbox-height: 22px;
+$form-checkbox-width: 22px;
+
+$form-checkbox-label-padding-left: 30px;
+$form-checkbox-label-after-border-width: 0 0 3px 3px;
+$form-checkbox-label-after-height: 7px;
+$form-checkbox-label-after-x-pos: 5px;
+$form-checkbox-label-after-y-pos: 6px;
+$form-checkbox-label-after-width: 12px;
+
+$form-radio-height: 22px;
+$form-radio-width: 22px;
+
+$form-radio-label-padding-left: 30px;
+$form-radio-label-after-border-width: 6px;
+$form-radio-label-after-x-pos: 5px;
+$form-radio-label-after-y-pos: 5px;
+
+$form-element-padding-horizontal: 5px;
+$form-element-padding-vertical: 5px;
```

Comment:
Probably better if these were specified in `rem`. Favour `rem` over `px` when defining variables.